src/commands/deploy/deploy.ts

```diff
@@ -63,6 +66,10 @@ async function deploy(actionContext: IActionContext, arg1: vscode.Uri | string |
         validateGlobSettings(context, context.effectiveDeployFsPath);
     }
 
+    if (language === ProjectLanguage.CSharp) {
```

Comment:
I believe the bitness can only be changed on Windows function apps, right? If so, you should also check
`&& !node.root.client.isLinux`

Reply:
Acknowledged